vault/dendron.ref.commands.md

```diff
@@ -495,6 +495,11 @@ Right Click on the vault > Dendron: Vault Remove
 
 ![[dendron://dendron.dendron-site/dendron.topic.vaults#vault-convert,1:#*]]
 
+### Migrate to Self Contained Vault
+
+Converts an existing vault into a [[self contained vault|dendron.topic.vaults.self-contained]].
```

Comment:
callout that this only works for regular vaults

also callout that this will result in multiple dendron.yml file if your workspace only has a single vault